clients/src/main/java/org/apache/kafka/common/config/ConfigDef.java

```diff
@@ -705,7 +710,7 @@ else if (value instanceof String)
                         if (trimmed.isEmpty())
                             return Collections.emptyList();
                         else
-                            return Arrays.asList(trimmed.split("\\s*,\\s*", -1));
+                            return Arrays.asList(SPLIT_COMMA_WITH_WHITESPACE.split(trimmed, -1));
```

Comment:
Is it necessary to add SPLIT_ and REPLACE_ to the names of patterns? The operation performed with the pattern is not part of it. In theory one pattern could be used to both split and replace.

Reply:
Good point.